extensions/dicom-sr/src/OHIFCornerstoneSRViewport.js

```diff
@@ -81,25 +92,89 @@ function OHIFCornerstoneSRViewport({
     ] = useTrackedMeasurements();
   }
 
+  // Locked if tracking any series
+  let isLocked = trackedMeasurements?.context?.trackedSeries?.length > 0;
+  useEffect(() => {
+    isLocked = trackedMeasurements?.context?.trackedSeries?.length > 0;
+  }, [trackedMeasurements]);
+
+  function _getToolAlias() {
+    const primaryToolId = ToolBarService.state.primaryToolId;
+    let toolAlias = primaryToolId;
+
+    switch (primaryToolId) {
```

Comment:
I see why you've done this, it feels nasty but I can't think of a better idea. You could add some meta to the tool's configuration and key off that, but that's not much different.